src/fidesops/ops/graph/config.py

```diff
@@ -305,9 +305,10 @@ class ObjectField(Field):
 
     fields: Dict[str, Field]
 
+    @classmethod
     @validator("data_categories")
```

Comment:
`@classmethod` should be on bottom. Decorators run bottom to top, this almost never matters, but for some reason unknown to me pydantic seems to care about this.

```suggestion
    @validator("data_categories")
    @classmethod
```

Reply:
thanks @sanders41 ! I'll give that a try